README.md

```diff
@@ -1,5 +1,7 @@
 # go-expect
 
+![Go](https://github.com/leozz37/go-expect/workflows/Go/badge.svg)
+[![codecov](https://codecov.io/gh/leozz37/go-expect/branch/master/graph/badge.svg?token=ZV4DeDXZ7g)](https://codecov.io/gh/leozz37/go-expect)
```

Comment:
These links seem to pointing to your fork instead of the Netflix org.

Reply:
I removed the codecov badge because the repo owner needs to activate the repo on [their website](codecov.io) for it to work.